src/cmd/src/frontend.rs

```diff
@@ -249,6 +249,10 @@ impl StartCommand {
         .await
         .context(StartFrontendSnafu)?;
 
+        instance
+            .build_system_metric_task(&opts.system_metric)
```

Comment:
If we rename the `system_metric` to `export_metric`, then we can rename this function to `build_export_metrics_task`, it looks better.